client/src/file/api/mod.rs

```diff
@@ -329,6 +329,10 @@ impl TryFrom<&[u8]> for Keyring {
     }
 }
 
+fn salt() -> Vec<u8> {
```

Comment:
I don't think it is necessary to add this function

Reply:
Removed that commit.